format_test.go

```diff
@@ -80,6 +82,22 @@ var formatTests = []struct {
 	about: "slice",
 	value: []int{1, 2, 3},
 	want:  "[]int{1, 2, 3}",
+}, {
+	about: "bytes",
+	value: []byte("hello"),
+	want:  `[]uint8("hello")`,
```

Comment:
Agreed in principle, but it would be an intrusive change. `Printf("%T")` prints `[]uint8` and the pretty package also prints `[]uint8`, so it's fairly consistent at least currently. I don't think it's worth doing now. If/when we replace `kr/pretty` would be a good time to do it, I think.